content/build/guides/styling.md

```diff
@@ -0,0 +1,109 @@
+---
+
+<meta>Needs review</meta>
+
+Styling in Elm is an exciting blend of functional programming and web development that offers several approaches to achieve visually appealing and responsive designs. Elm, known for its strong emphasis on simplicity, reliability, and maintainability, provides different packages to handle styling needs, catering to various preferences and workflows. Whether you're comfortable with traditional HTML and CSS, or looking for more type-safe or declarative approaches, Elm has you covered. The primary packages used for styling in Elm are `elm/html`, `rtfeldman/elm-css`, and `mdgriffith/elm-ui`. Each of these packages offers unique features and benefits, allowing you to choose the one that best fits your project's requirements and your personal coding style. By leveraging Elm's powerful type system and functional programming paradigm, you can create styles that are both robust and easy to maintain.
+
+## elm/html
+The [elm-html](https://package.elm-lang.org/packages/elm/html/latest/) package allows you to write HTML directly in Elm. It provides a way to create HTML elements and attributes using Elm's functional programming paradigm. This package is backed by `elm/virtual-dom`, which handles the rendering of DOM nodes efficiently.
+
+**Example:**
+```elm
+module Main exposing (..)
+
+import Html exposing (Html, div, text)
+import Html.Attributes exposing (style)
+
+
+main =
+    div
+        [ style "display" "flex"
+        , style "justify-content" "center"
+        , style "align-items" "center"
+        , style "height" "100vh"
+        ]
+        [ text "Centered Element" ]
+```
+
+**Pros:**
+- Directly integrates with HTML, making it easy to use for those familiar with HTML and CSS.
+- Efficient rendering with `elm/virtual-dom`.
+- Flexibility to use any CSS properties.
+
+**Cons:**
+- Requires manual handling of CSS, which can lead to inconsistencies.
+- Limited abstraction compared to other packages.
+
+## rtfeldman/elm-css
+The [elm-css](https://package.elm-lang.org/packages/rtfeldman/elm-css/latest/) package provides a way to write CSS in Elm with type safety. It allows you to define CSS styles using Elm's type system, ensuring that your styles are valid and reducing runtime errors. This package is inspired by CSS preprocessors like Sass and styled-components.
+
+**Example:**
+```elm
+module Main exposing (..)
+
+import Css exposing (..)
+import Html.Styled exposing (Html, div, text, toUnstyled)
+import Html.Styled.Attributes exposing (css)
+
+
+main =
+    toUnstyled <|
+        div
+            [ css
+                [ displayFlex
+                , justifyContent center
+                , alignItems center
+                , height (vh 100)
+                ]
+            ]
+            [ text "Centered Element" ]
+```
+
+**Pros:**
+- Type-safe CSS, reducing runtime errors.
+- Reusable styles and mixins.
+- Integration with Elm's type system.
+
+**Cons:**
+- Requires learning the `elm-css` syntax.
```

Comment:
```suggestion
- Requires learning the set of `elm-css` functions.
```
There's no syntax to learn, it's just function calls. But I agree there's a bit to learn.